anta/tests/logging.py

```diff
@@ -282,7 +282,7 @@ class VerifyLoggingTimestamp(AntaTest):
     def test(self) -> None:
         """Main test function for VerifyLoggingTimestamp."""
         log_pattern = r"ANTA VerifyLoggingTimestamp validation"
-        timestamp_pattern = r"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{6}-\d{2}:\d{2}"
+        timestamp_pattern = r"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{6}[\+-]\d{2}:\d{2}"
```

Comment:
quick question - any reason you have added `\` in the list of accepted character to escape `+`?
```suggestion
        timestamp_pattern = r"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{6}[\+-]\d{2}:\d{2}"
```

Trying out here: https://regex101.com/r/WHhYDT/1 it seems that the result would be the same with and without?

Otherwise LGTM

Reply:
hi @gmuloc I escaped it thinking it was a protected character:

_"+ matches the previous token between one and unlimited times, as many"_

having said that, once it is inside the brackets looks like it treats it like a literal as you say. I'll update and push out again.

Reply:
Pushed the revised code after testing, @gmuloc feel free to review and thanks for showing me, learned something new

Reply:
thanks both were working but just wanted to make sure if there was a reason or not!